crates/iota-protocol-config/src/lib.rs

```diff
@@ -600,8 +600,8 @@ pub struct ProtocolConfig {
     // Base gas price for computation gas, nanos per computation unit.
     base_gas_price: Option<u64>,
 
-    /// The number of tokens minted for validator re
-    validator_target_reward: Option<u64>,
+    /// The number of tokens minted for validator per epoch.
+    validator_subsidy: Option<u64>,
```

Comment:
We can't change names of existing protocol config fields. We can only add new fields and change values. I will revert this.